src/format.ts

```diff
@@ -402,6 +402,15 @@ export function createFormat<T>(model: Model, type: any, format: string): Format
 			nullFormat = formats.length > 2 ? formats[2] : "";
 		}
 
+		/**
+		 * Determines if the given string matches the given format option
+		 * @param str The text to check
+		 * @param formatValue The true or false format option
+		 */
+		const isFormatMatch = (str, formatValue) => {
+			return str.trim().toLowerCase() === formatValue.trim().toLowerCase();
+		};
```

Comment:
The main fix of the PR is to convert True/False values to lower-case before comparing to the input. I also added trimming of the input text, even though its being done in the `Format` class itself, since I'm not sure it necessarily should be done there. Finally, moved this logic into a reusable function.